src/proto/flwr/proto/exec.proto

```diff
@@ -28,6 +28,7 @@ service Exec {
 message StartRunRequest {
   bytes fab_file = 1;
   map<string, string> override_config = 2;
+  map<string, string> options_config = 3;
```

Comment:
How about:
```suggestion
  map<string, string> federation_options = 3;
```

Reply:
Or `federation_config` to be consistent with `override_config`?

Reply:
`federation_config` seems good